.github/workflows/extension-ci.yml

```
@@ -0,0 +1,203 @@
+        env:
+          CI: true
+          FORCE_COLOR: 1
+        timeout-minutes: 10
+
+      - name: Upload Test Results
+        if: always()
+        uses: actions/upload-artifact@v4
+        with:
+          name: extension-test-results
+          path: apps/extension/test-results
+          retention-days: 30 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)